frontend/src/components/smart/WeaponGrid.vue

```diff
@@ -293,4 +293,17 @@ export default Vue.extend({
     margin-top : 10px;
   }
 }
+
+/* Needed to adjust weapon list */
+@media all and (max-width:  767.98px) {
```

Comment:
do we really need a media query? can this be done with bootstrap?

Reply:
Bootstrap's functionality is mainly the responsiveness of the elements,  mainly implemented thru cols which are mainly setting the width of divs. I've already tried to not do media queries since you didnt like them but these changes are not about arrangement but about the properties of individual elements which Bootstrap as far as I know don't manipulate directly.

Reply:
If you want we can use JQuery to reassign classes to the  individual elements.  We will have to load a bunch of  JS for  each affected element.   One caveat to is since we will have to check screen size at some point (whether from an event handler or at  the beginning) we will have to deal with as I'm sure you know  is the JS issue of always having to check if elements exist when they are first loaded.  I'm also not  sure  of the  performance impact of  that much JS might have...

Reply:
jquery is off the table.

Reply:
ok...